index.html

```diff
@@ -175,39 +192,45 @@ <h1>About me</h1>
       <button class="get-myresume-btn" type='button'>Get my resume</button>
     </div>
     <div class="competence">
-      <ul class="competence-list">
-        <li class="list-title languages">
-          Languages
-        </li>
-      </ul>
-      <ul class="competence-list">
-        <li class="competence-item">Javascript</li>
-        <li class="competence-item">Ruby</li>
-        <li class="competence-item">Css</li>
-        <li class="competence-item">Html</li>
-      </ul>
-      <ul class="competence-list">
-        <li class="list-title frameworks">
-          Frameworks
-        </li>
-      </ul>
-      <ul class="competence-list">
-        <li class="competence-item">Bootstrap</li>
-        <li class="competence-item">React</li>
-        <li class="competence-item">Gatsby</li>
-        <li class="competence-item">Laravel</li>
-      </ul>
-      <ul class="competence-list">
-        <li class="list-title skills">
-          Skills
-        </li>
-      </ul>
-      <ul class="competence-list">
-        <li class="competence-item">Github</li>
-        <li class="competence-item">Codepen</li>
-        <li class="competence-item">Codesandbox</li>
-        <li class="competence-item">Replit</li>
-      </ul>
+      <div class="languajes-div">
+        <ul class="competence-list">
+          <li class="list-title languages">
+            Languages
+          </li>
+        </ul>
+        <ul class="competence-list">
+          <li class="competence-item">Javascript</li>
+          <li class="competence-item">Ruby</li>
+          <li class="competence-item">Css</li>
+          <li class="competence-item">Html</li>
+        </ul>
+      </div>
+      <div class="frameworks-div">
+        <ul class="competence-list">
+          <li class="list-title frameworks">
+            Frameworks
+          </li>
+        </ul>
+        <ul class="competence-list">
+          <li class="competence-item">Bootstrap</li>
+          <li class="competence-item">React</li>
```

Comment:
- [ ] The Language, Frameworks and skills list should be left aligned on desktop version while they are center aligned on mobile. Kindly update as in the Figma design.
#### Design
![language-and-skills-section-left-figma](https://user-images.githubusercontent.com/44057863/167276297-06b38dfc-852b-4b40-8532-5d10890677d6.png)
#### Imp
![language-and-skills-section-left-impl](https://user-images.githubusercontent.com/44057863/167276304-bedf8041-679d-498e-ba19-24240fc32742.png)
l

Reply:
Thanks.

Reply:
Done